src/modules/assessments/components/AssessmentHistory.tsx

```diff
@@ -0,0 +1,33 @@
+import { CommonLabeledTextBlock } from '@/components/elements/CommonLabeledTextBlock';
+import {
+  formatDateTimeForDisplay,
+  parseHmisDateString,
+} from '@/modules/hmis/hmisUtil';
+import { UserFieldsFragment } from '@/types/gqlTypes';
+
+const AssessmentHistoryInfo: React.FC<{
+  label: string;
+  user?: UserFieldsFragment | null;
+  date?: string | null;
+}> = ({ label, user, date }) => {
+  const fmtDate = parseHmisDateString(date);
+  let content: string | null = null;
+  if (fmtDate && user) {
+    content = `${user.name} at ${formatDateTimeForDisplay(fmtDate)} `;
+  } else if (user) {
+    // this case probably doesn't occur in practice, we should always have dates
+    content = user.name;
+  } else if (fmtDate) {
+    content = `System User at ${formatDateTimeForDisplay(fmtDate)}`;
```

Comment:
I think 'system user' may be most accurate here, if we don't have a record of the user, it was probably created via the initial data import. Perhaps its better to just say 'Unknown User'

Reply:
Agree, lets change to unknown user